README.rst

```diff
@@ -28,6 +28,10 @@ It comes with a GNU Lesser General Public License of version 3, which
 means that it can be modified and freely distributed, although we take
 no responsibility for its misuse.
 
+For more information, have a look at the documentation_!
+
+.. _documentation: https://cosmo-epfl.github.io/librascal/
```

Comment:
FYI this line is included in the built documentation, so it's now self-referencing. Douglas Hofstadter would approve.